### DISCUSSION TITLE
No categories or labels in "Teams" discussions.

### DISCUSSION THREAD

**Discussion by abalter** [community/community#49107] [Discussions] [answered]
### Select Topic Area

Bug

### Body

I don't know if this is a bug or "feature."

Discusions for repos and organizations look like [this](https://docs.github.com/en/discussions/managing-discussions-for-your-community/managing-discussions#pinning-a-discussion)

![image](https://user-images.githubusercontent.com/5349876/222810933-c325c288-f911-4afb-b6e4-63e0f994ab9b.png)

However the discussion page I created for a Team looks like this:

![image](https://user-images.githubusercontent.com/5349876/222811176-eb7d3cf8-0d44-4372-b27e-d99b122a15a8.png)

No labels, no categories, no banner, etc.

Is this by design? I would really like the full discussion platform for my team. Is there a switch I need to flip?

**Comment by abalter** [comment, marked as answer]
I just noticed that there is info on "Teams Discussions" which are [going to sunset in May 2023](https://docs.github.com/en/rest/teams/discussions?apiVersion=2022-11-28). 

So can close this.

**Reply by rungrotArunon** [reply]
thank you
